ch_backup/backup/metadata/table_metadata.py

```diff
@@ -53,6 +55,24 @@ def get_parts(self, *, excluded_parts: Set[str] = None) -> List[PartMetadata]:
                     PartMetadata.load(self.database, self.name, part_name, raw_metadata)
                 )
 
+        if self.engine == REPLACING_MERGE_TREE:
+
+            def split_part_name(part):
+                chunks = part.split("_", maxsplit=3)
+                partition = ""
+                suffix = ""
+                try:
+                    partition = chunks[0]
+                    start_range = int(chunks[1])
+                    finish_range = int(chunks[2])
+                    suffix = chunks[3]
+                except (IndexError, ValueError):
+                    start_range = 0
+                    finish_range = 0
+                return partition, start_range, finish_range, suffix
```

Comment:
Let's use `from typing import NamedTuple` for clarity. It is also lexicographically comparable as a usual tuple

Reply:
Ok